### ISSUE TITLE
Implement GPU code for Kraskov nearest neighbour search

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jlizier/jidt#15]
```
Some tips on using aparapi for java here (suggested by Michael Wibral):

http://blogs.amd.com/developer/2011/09/14/i-dont-always-write-gpu-code-in-java-b
ut-when-i-do-i-like-to-use-aparapi/

```

Original issue reported on code.google.com by `joseph.lizier` on 7 Apr 2013 at 11:08

**Comment by pmediano** [comment]
A pull request has been submitted: #55

**Comment by jlizier** [comment]
Quick note on progress: GPU functionality included in release v1.4 for KSG MI; documentation and sample code to come. Conditional MI still to come also.
